packages/babel-traverse/src/path/family.ts

```diff
@@ -312,6 +312,53 @@ export function getAllPrevSiblings(this: NodePath): NodePath[] {
   return siblings;
 }
 
+type digitalWithoutZero = "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9";
+type digital = "0" | digitalWithoutZero;
+
+// only support 0 to 999 (since we won't use too large number in path.get())
+type Num =
+  | `${digital}`
+  | `${digitalWithoutZero}${digital}`
+  | `${digitalWithoutZero}${digital}${digital}`;
+
+// convert "1" to 0 (string index to number index)
+type ToNumber<T extends string> = T extends Num ? 0 : T;
```

Comment:
From this name I would expect the output to always be a number, and for it to be the number corresponding to the string (e.g. `ToNumber<"3">` is `3`).
Maybe we could name it `MaybeToIndex`, to better explain what it does, and make it just return `number` rather than a specific one?